ext/rdoc/extconf.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Generate all parse files using racc and kpeg. This is not necessary for regular gem installation, but it is when
+# installing RDoc from the git source. Without this, the generated parse files would not exist and RDoc would not work
+system("rake generate") if Dir.exist?(File.join("..", "..", ".git"))
```

Comment:
`.git` may be a regular file, when using `--git-dir` option.
```suggestion
system("rake generate") if File.exist?("../../.git")
```